mappings/net/minecraft/world/dimension/DimensionType.mapping

```diff
@@ -5,7 +5,7 @@ CLASS net/minecraft/class_2874 net/minecraft/world/dimension/DimensionType
 	FIELD field_13077 suffix Ljava/lang/String;
 	FIELD field_13078 THE_END Lnet/minecraft/class_2874;
 	FIELD field_20658 biomeAccessType Lnet/minecraft/class_4545;
-	FIELD field_24504 hasCeiling Z
+	FIELD field_24504 hellish Z
```

Comment:
Why? `hasCeiling` is referenced in a string, and `hellish` is a name directly related to the Nether

Reply:
See the comment from Sollace: https://github.com/FabricMC/yarn/pull/1353#issuecomment-628907223

I wasn't sure which to keep.